standard/stream_links.lua

```diff
@@ -93,11 +95,100 @@ function StreamLinks.processStreams(forwardedInputArgs)
 
 				streamValue = StreamLinks.resolve(lookUpPlatform, streamValue)
 			end
+
+			if FeatureFlag.get('new_stream_format') then
+				local key = StreamLinks.StreamKey(platformName):toString()
+				streams[key] = streamValue
+			end
 			streams[platformName] = streamValue
 		end
 	end
 
 	return streams
 end
 
+--- StreamKey Class
+-- Contains the truple that makes up a stream key
+-- [platform, languageCode, index]
+StreamLinks.StreamKey = Class.new(
+	function (self,...)
+		self:new(...)
+	end
+)
+local StreamKey = StreamLinks.StreamKey
+
+function StreamKey:new(tbl, languageCode, index)
+	local platform
+	-- Input is another StreamKey - Make a copy
+	if StreamKey._isStreamKey(tbl) then
+		platform = tbl.platform
+		languageCode = tbl.languageCode
+		index = tbl.index
+	-- Input is another table, assume format is {platform, languageCode, index}
+	elseif type(tbl) == 'table' then
+		platform, languageCode, index = unpack(tbl)
```

Comment:
do we actually want to allow that?
(not necessarily against ist, just asking)

Reply:
Sure let's drop it